internal/impl/confluent/processor_schema_registry_encode.go

```diff
@@ -348,26 +348,33 @@ func (s *schemaRegistryEncoder) getLatestEncoder(subject string) (schemaEncoder,
 	}
 
 	var codec *goavro.Codec
-	if codec, err = goavro.NewCodecForStandardJSON(resPayload.Schema); err != nil {
-		s.logger.Errorf("failed to parse response for schema subject '%v': %v", subject, err)
-		return nil, 0, err
+	if s.avroRawJSON {
+		if codec, err = goavro.NewCodecForStandardJSON(resPayload.Schema); err != nil {
+			s.logger.Errorf("failed to parse response for schema subject '%v': %v", subject, err)
+			return nil, 0, err
+		}
+	} else {
+		if codec, err = goavro.NewCodec(resPayload.Schema); err != nil {
+			s.logger.Errorf("failed to parse response for schema subject '%v': %v", subject, err)
+			return nil, 0, err
+		}
 	}
 
+	s.logger.Tracef("codec is: %#v", codec)
```

Comment:
please let me know a good format for trace logs. I think it would be nice to expose the user what codec has been parsed if trace log level is enabled.